sem-service-check-params

```diff
@@ -75,7 +72,47 @@ params-check::postgres () {
     [ "$dbname" ] && db=" -e POSTGRES_DB=$dbname"
   fi
   default_params="$default_params$username$password$db "
-  echo "${version_map[$service_version]} $default_params"
+  echo "$service_version $default_params"
+}
+
+params-check::postgis () {
+  default_params='--net=host --rm -d --name postgis -v /var/run/postgresql:/var/run/postgresql '
+  service_version="9.6-2.5"
+  sudo mkdir -p /var/run/postgresql ; sudo chmod -R 0777 /var/run/postgresql
```

Comment:
Use two lines instead of pushing two commands into one line with `;`

Reply:
Fixed in all 3 places.